### PR TITLE
add support for kafka endpoint configuration

### PR DESCRIPTION
**Pull request by melindaloubser1**
- add support for configuring kafka as event broker in endpoints
- closes https://github.com/RasaHQ/rasa-x-helm/issues/207

### CODE REVIEW

**Comment by melindaloubser1** [comment]
I only realized after adding this that it's not necessary; you can just configure `additionalEndpoints` for kafka.
https://github.com/RasaHQ/rasa-x-helm/issues/207#issuecomment-913489694

I'd suggest closing this and the related issue since it's a bit confusing to have a "kafka" section that really only pertains to to the `rasa` deployment's endpoints entries. We could add a note in the README instead.

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml [hunk 1]
```diff
@@ -48,6 +48,10 @@ data:
       {{- end }}
       {{- end }}
     {{- end }}
+    {{- if $.Values.kafka.enabled }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/rasa-config-files-configmap.yaml:51]: Can we add `fail` to the `NOTES.txt` if `rabbitmq.enabled=true` and `kafka.enabled=true` - something like we have in the rasa helm chart - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L42-L45

**Review by tczekajlo — CHANGES_REQUESTED** [review]
One small suggestion.

Can you bump the chart version and add a changelog entry as well? - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L42-L45

**Comment by tczekajlo** [comment]
> I only realized after adding this that it's not necessary; you can just configure `additionalEndpoints` for kafka.
> [#207 (comment)](https://github.com/RasaHQ/rasa-x-helm/issues/207#issuecomment-913489694)
> 
> I'd suggest closing this and the related issue since it's a bit confusing to have a "kafka" section that really only pertains to to the `rasa` deployment's endpoints entries. We could add a note in the README instead.

Good point, it can be done by using the rasa helm chart.

**Pull request closed without merging**